### ISSUE TITLE
Icon-request: microsoft-azure

### ISSUE THREAD

**Issue by djagtap** [FortAwesome/Font-Awesome#5212]
Reference: http://blog.geuer-pollmann.de/pages/azure-iconography-reference.html
Icon with o letter represents MS Azure service

**Comment by graymatter00** [comment]
👍

**Comment by flyboarder** [comment]
+1 All cloud vendors would be nice

**Comment by Palpie** [comment]
:+1:

**Comment by pratnala** [comment]
Azure logo was updated recently. +1 to that

**Comment by morgzz** [comment]
+1

**Comment by qJake** [comment]
I accidentally opened a duplicate of this awhile back - #14072

Can we get the likes from that one merged into this one, and maybe then this will finally show up on the leaderboard? 😄

**Comment by DaSchTour** [comment]
Can somebody please add the brand icon label to make this visible on the request page of fontawesome

**Comment by qJake** [comment]
Pinging an active project member: @tagliala can you fix the title for us please? This is a brand icon request! 😃

**Comment by thecherrytree** [comment]
@tagliala can you direct us to an active member who can fix this?

**Comment by vpolimenov** [comment]
is there any progress around that ?

**Comment by DaSchTour** [comment]
I guess as there are still no labels added, this issue didn't get any attention of member of the font awesome team

**Comment by qJake** [comment]
Well @tagliala is active, I don't know how else to draw attention to this besides mentioning them?

**Comment by Jack-Barry** [comment]
Would be nice to have the Azure logo to differentiate between Azure and Microsoft applications

**Comment by dimme** [comment]
Has this been included yet? It would be very nice.

**Comment by qJake** [comment]
Pinging @tagliala again... can you please fix this so it's a proper request and tracked correctly?

Thanks! 😄

**Comment by sandorfr** [comment]
It's a bit silly that the faster growing cloud provider doesn't have its Icon in FA :D

**Comment by MJunaidAhmad** [comment]
(y)

**Comment by johnmckeever** [comment]
I'd be very grateful if you could take a look at this request.

**Comment by tagliala** [comment]
@johnmckeever thanks for the heads-up

Is #14058 a duplicate of this request?

**Comment by johnmckeever** [comment]
I don't think so, @tagliala. Microsoft Azure (the platform) and  Microsoft Azure DevOps (the product, which has both SaaS and On-Premises offerings) have different icons.

### Azure
![image](https://user-images.githubusercontent.com/10860815/86916792-76f85580-c167-11ea-89ba-afeccc122a16.png)

### Azure DevOps
![image](https://user-images.githubusercontent.com/10860815/86916754-6b0c9380-c167-11ea-901b-78571ad4065d.png)

**Comment by tagliala** [comment]
Thanks!

I can see that I've been mentioned a lot of time here during the years but I didn't notice before.

Apologies for the missing label, now it is fixed and I can see this icon at position `42` of our Brand leaderboard.

Feel free to ask developers to 👍 this request (by using the reaction, not +1s comments) 

You may also be interested in taking a look at our [Brand Leaderboard FAQs](https://fontawesome.com/community/leaderboard/brands#faqs), they contain a lot of information about how requests work

**Comment by qJake** [comment]
Yay! Thank you @tagliala. Fingers crossed we can get enough votes to get the Azure icon implemented.

**Comment by bert-laverman** [comment]
👍

**Comment by thecherrytree** [comment]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

**Comment by AJarjis** [comment]
👍

**Comment by Isitar** [comment]
Azure recently changed his logo:

https://azure.microsoft.com/en-us/blog/a-fluent-new-look-for-the-azure-icon/

**Comment by pratnala** [comment]
Azure updated its logo twice and we still don't have an icon

**Comment by PeripheralMike** [comment]
👍

**Comment by ebioux-btig** [comment]
👍🏻

**Comment by stannum-l** [comment]
:+1:

**Comment by RaysceneNS** [comment]
👍

**Comment by Vilican** [comment]
👍

**Comment by dileepmt** [comment]
👍

**Comment by Serhii-Tsypivko** [comment]
👍

**Comment by thecherrytree** [comment]
👍

**Comment by rikdeakin** [comment]
👍🏻

**Comment by thecherrytree** [comment]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

**Comment by thecherrytree** [comment]
> I accidentally opened a duplicate of this awhile back - #14072
> 
> Can we get the likes from that one merged into this one, and maybe then this will finally show up on the leaderboard? 😄

@qJake, we're getting old. Wont be writing any software or using azure cloud by the time this gets added to font-awesome.

**Comment by CSharpFiasco** [comment]
Are there any roadblocks that the community can help with?

**Comment by qJake** [comment]
> > I accidentally opened a duplicate of this awhile back - #14072
> > Can we get the likes from that one merged into this one, and maybe then this will finally show up on the leaderboard? 😄
> 
> @qJake, we're getting old. Wont be writing any software or using azure cloud by the time this gets added to font-awesome.

When I originally opened #14072, one of my kids wasn't even born yet. They will now be going to kindergarten in a few months.

I think the only hope we have for an Azure icon is if they can graduate college, get a job at Font Awesome, and then add it for us. 🤣

**Comment by TheLe0** [comment]
👍

**Comment by JaeTLDR** [comment]
9 years and still nothing?

**Comment by Banner-Keith** [comment]
Seriously. We pay for pro. It'd be nice if we could have some of our money go to adding icons we ask for.

**Comment by ghost** [comment]
Trying to add the Azure icon and it's still not there.  weird that the cloud has been around awhile and still no icon.  9 years since the original request.

**Comment by Jack-Barry** [comment]
I'm beginning to think somebody at FA just does not like Azure... the Twitter rebrand logo was opened and closed in 9 days vs. 9 years and running for this one

I don't even use FA anymore, just get pinged with this running joke every so often

**Comment by victorfrye** [comment]
Came here hoping to add a request for Azure and find out there's one so old the Azure icon has changed twice in the lifetime of this issue... Wow.

**Comment by ugwulo** [comment]
2024 and yet nothing. Weird!

**Comment by BSG2000** [comment]
I completely agree! Having an Azure brand logo in Font Awesome would be a good addition. It would not only represent the diversity of cloud services but also enhance usability for developers incorporating Azure into their projects. An official logo would improve visual consistency and professionalism in the representation of Azure services.

**Comment by AndNovian** [comment]
+1 please

**Comment by qJake** [comment]
Meanwhile, halfway through 2025, and we have an icon for AWS, as well as 9 different Google services... yet no Azure icon.

**Comment by JaeTLDR** [comment]
> Meanwhile, halfway through 2025, and we have an icon for AWS, as well as 9 different Google services... yet no Azure icon.

Not only do we have countless Google and Amazon properties we have multiple fictional companies before one of the top 3 Cloud providers (Why do we need 3 silicon valley icons?). 

FortAwesome, do you have some kind of deal with someone? or is it is Microsoft annoyed you?, either way this is completley ridiculous .

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
